docs/cluster-provisioning.md

```diff
@@ -207,6 +213,11 @@ ProvisioningRequestValidated -> ClusterInstanceRendered -> ClusterResourcesCreat
         reason: Completed
         status: "True"
         type: HardwareProvisioned
+      - lastTransitionTime: "2024-10-19T00:27:30Z"
+        message: Node configuration has been applied to the ClusterInstance
```

Comment:
Should the message be "Node configuration has been applied to the `rendered` ClusterInstance" based on the code?

Reply:
Changed.